content/en/docs/languages/csharp/basics.md

```diff
@@ -305,7 +305,7 @@ so that clients can actually use our service. The following snippet shows how we
 do this for our `RouteGuide` service:
 
 ```csharp
-var features = RouteGuideUtil.ParseFeatures(RouteGuideUtil.DefaultFeaturesFile);
+ var features = RouteGuideUtil.LoadFeatures();
```

Comment:
This line seems to have been indented by mistake. Please remove the extra leading whitespace.